Package.swift

```diff
@@ -4,10 +4,10 @@ import PackageDescription
 let package = Package(
   name: "StencilSwiftKit",
   products: [
-      .library(name: "StencilSwiftKit", targets: ["StencilSwiftKit"])
+    .library(name: "StencilSwiftKit", targets: ["StencilSwiftKit"])
   ],
   dependencies: [
-      .package(url: "https://github.com/stencilproject/Stencil.git", .upToNextMinor(from: "0.13.0"))
+    .package(url: "https://github.com/stencilproject/Stencil.git", .upToNextMajor(from: "0.14.0"))
```

Comment:
When updating the Package.swift`, you need to also [update the dependency in the `StencilSwiftKit.podspec` too](https://github.com/SwiftGen/StencilSwiftKit/blob/stable/StencilSwiftKit.podspec#L30)

_(one day we'll configure Danger on this repo too similar to how we have it in the main SwiftGen repo, and Danger would automatically warn you about that…)_

Once you've updated the `.podspec`, don't forget to run `bundle exec pod install` to update the project's `Podfile.lock` accordingly and commit the change to the lockfile too.